workloads/router-perf-v2/common.sh

```diff
@@ -36,15 +38,24 @@ deploy_infra(){
   else
     ${ENGINE} run --rm -v $(pwd)/templates:/templates:z -v ${KUBECONFIG}:/root/.kube/config:z -v $(pwd)/${INFRA_CONFIG}:/http-perf.yml:z ${KUBE_BURNER_IMAGE} init -c http-perf.yml --uuid=${UUID}
   fi
+
+  log "Creating configmap from workload.py file"
   oc create configmap -n http-scale-client workload --from-file=workload.py
   log "Adding workload.py to the client pod"
-  oc set volumes -n http-scale-client deploy/http-scale-client --type=configmap --mount-path=/workload --configmap-name=workload --add
+  oc set volumes -n http-scale-client deploy/http-scale-client --add --type=configmap --mount-path=/workload --configmap-name=workload
+  log "Waiting for http-scale-client rollout"
   oc rollout status -n http-scale-client deploy/http-scale-client
 }
 
 tune_liveness_probe(){
   log "Disabling cluster version and ingress operators"
-  oc scale --replicas=0 -n openshift-cluster-version deploy/cluster-version-operator
+  oc patch clusterversions/version --type=json --patch='[{"op":"add","path":"/spec/overrides","value":[{"kind":"Deployment","group":"apps/v1","name":"ingress-operator","namespace":"openshift-ingress-operator","unmanaged":true}]}]'
+  if [[ ! -z "${HAPROXY_IMAGE}" ]]; then
+    log "Replacing router with ${HAPROXY_IMAGE}"
+    oc -n openshift-ingress-operator patch deploy/ingress-operator --type=strategic --patch='{"spec":{"template":{"spec":{"containers":[{"name":"ingress-operator","env":[{"name":"IMAGE","value":"'${HAPROXY_IMAGE}'"}]}]}}}}'
+    log "Waiting 60s for ingress-operator IMAGE to change"
+    sleep 60
```

Comment:
Is there something we can "wait" for instead of a blind sleep?

Reply:
We can try lol